foodplanner/lib/pages/create_ingredient_page.dart

```diff
@@ -102,57 +102,24 @@ class _CreateIngredientPageState extends State<CreateIngredientPage> {
-                                  CupertinoDialogAction(
-                                    isDefaultAction:
-                                        true, // Highlight the default action.
-                                    onPressed: () async {
-                                      // Leads the user to the camera page.
-                                      Ingredient newIngredient = Ingredient(
-                                        // The new ingredient.
-                                        id: 0,
-                                        name: ingredientNameController
-                                            .text, // The name recieved from the controller.
-                                        foodImageId: null,
-                                      );
-                                      newIngredient = Ingredient.fromJson(
-                                        jsonDecode(
-                                          (await ingredientServices
-                                                  .createIngredient(
-                                            client!,
-                                            AuthProvider(),
-                                            newIngredient.name,
-                                            newIngredient.foodImageId,
-                                          ))
-                                              .body,
-                                        ),
-                                      );
-                                      Navigator.pop(context, null);
-                                      Navigator.pop(context, newIngredient);
-                                    },
-                                    child: const Text("Ja"),
-                                  ),
-                                  CupertinoDialogAction(
-                                    isDestructiveAction: true,
-                                    onPressed: () {
-                                      Navigator.pop(context);
-                                    },
-                                    child: const Text('Nej'),
-                                  ),
-                                ],
+                        ? () async {
+                            Ingredient newIngredient = Ingredient(
+                              id: 0,
+                              name: ingredientNameController.text,
+                              foodImageId: null,
+                            );
+                            newIngredient = Ingredient.fromJson(
+                              jsonDecode(
+                                (await ingredientServices.createIngredient(
+                                  client!,
+                                  AuthProvider(),
+                                  newIngredient.name,
+                                  newIngredient.foodImageId,
+                                ))
+                                    .body,
```

Comment:
Couldn't find any issues with the implementation, so instead I'll nitpick on formatting :rocket: 
```suggestion
                                )).body,
```